### ISSUE TITLE
Server Startup Issue (no crash, per se...)

### ISSUE THREAD

**Issue by Caldoric** [vectorwing/FarmersDelight#891]
### Minecraft version

1.20.1

### Farmer's Delight version

1.2.4

### Forge version

47.2.20

### Description

trying to get a large assortment of mods onto a server I'm putting together, and have hit a wall on the latest bundle of mods I pasted in. the issue isn't generating a crash report, since the server itself apparently never fully gets up and running before the issue crops up. nonetheless, attaching contents of latest server startup log to this report (several sections below).

I believe this is the portion of interest (just above the end of log):
v~~~~v
[main/ERROR] [ne.mi.ev.EventBus/EVENTBUS]: Exception caught during firing event: vectorwing/farmersdelight/common/registry/ModEffects
	Index: 9
	Listeners:
		0: HIGHEST
		1: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@3a90e4f0
		2: NORMAL
		3: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@4f7c23e5
		4: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@16a664e
		5: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@4a317c28
		6: ASM: class net.bdew.lib.recipes.RecipeReloadListener tagsUpdated(Lnet/minecraftforge/event/TagsUpdatedEvent;)V
		7: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@773bb6fa
		8: ASM: net.minecraftforge.common.ForgeInternalHandler@3b24525d tagsUpdated(Lnet/minecraftforge/event/TagsUpdatedEvent;)V
		9: ASM: class cofh.thermal.core.common.event.TCoreCommonSetupEvents tagsUpdated(Lnet/minecraftforge/event/TagsUpdatedEvent;)V
		10: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@5976d0f3
		11: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@b7c9585
		12: ASM: class net.mehvahdjukaar.moonlight.forge.MoonlightForge onTagUpdated(Lnet/minecraftforge/event/TagsUpdatedEvent;)V
		13: ASM: com.blakebr0.cucumber.crafting.TagMapper@70785209 onTagsUpdated(Lnet/minecraftforge/event/TagsUpdatedEvent;)V
		14: LOWEST
		15: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@4aed78fe
		16: net.minecraftforge.eventbus.EventBus$$Lambda$4605/0x00007f69fcce4420@60cb9b3f

### Steps to reproduce

Attempted to start up the server, server failed to finish starting.

### Mod list

	- absentbydesign 1.8.0
	- ad_astra 1.15.18
	- ad_astra_giselle_addon 6.5
	- adaptive_performance_tweaks 11.0.0
	- adaptive_performance_tweaks_core 11.0.0
	- adaptive_performance_tweaks_gamerules 11.0.0
	- adaptive_performance_tweaks_items 11.0.0
	- adaptive_performance_tweaks_player 11.0.0
	- adaptive_performance_tweaks_spawn 11.0.0
	- adchimneys **.**.**.**
	- additional_lights 2.1.7
	- additionallanterns 1.1.1
	- adhooks **.**.**.**
	- adlods **.**.**.**
	- adpoles **.**.**.**
	- advancedperipherals 0.7.38r
	- advancementplaques 1.5.1
	- advgenerators **.**.**.**
	- ae2 15.0.23
	- ae2additions 5.0.5
	- ae2things 1.2.1
	- ae2wtlib 15.2.2-forge
	- aeinfinitybooster 1.20.1-1.0.0+21
	- aeroblender 1.20.1-1.0.1-neoforge
	- aether 1.20.1-1.3.1-neoforge
	- aether_emissivity 1.20.1-1.0.0-neoforge
	- aether_enhanced_extinguishing 1.20.1-1.0.0-neoforge
	- aetherdelight 1.0.0
	- aiimprovements 0.5.2
	- alchemistry 2.3.4
	- alchemylib 1.0.30
	- alexscaves 1.1.4
	- alexsdelight 1.5
	- alexsmobs 1.22.8
	- allthemodium 2.4.4
	- alltheores 2.2.3
	- allthetrims 3.4.2
	- allthewizardgear 1.20.1-1.1.1
	- almostunified 1.20.1-0.9.2
	- alternate_current 1.7.0
	- alwaysawitherskull 3.2
	- amplifiednether 1.2.4
	- another_furniture 1.20.1-3.0.1
	- anvilrestoration 2.2
	- apotheosis 7.3.4
	- apotheotic_additions 2.0.1
	- apotheoticcreation 1.0.0
	- apothiccurios 1.0.3b
	- appleskin 2.5.1+mc1.20.1
	- aquaculture 2.5.1
	- aquaculturedelight 1.0.0
	- architects_palette **.**.**.**
	- architectury 9.2.14
	- ars_artifice 1.20.1-2.0.4
	- ars_creo 4.1.0
	- ars_elemental 1.20.1-0.6.4.1
	- ars_nouveau 4.10.0
	- ars_ocultas 1.2.2
	- ars_scalaes 1.20.1-1.10.2
	- arseng 1.1.5
	- artifacts 9.3.0
	- astral_dimension 1.7.0
	- athena 3.1.2
	- atlaslib 1.1.10b
	- attributefix 21.0.4
	- attributeslib 1.3.4
	- azurelib 2.0.20
	- balm 7.2.2
	- bdlib **.**.**.**
	- betterchunkloading 1.20.1-3.3
	- betterfps 1.0.0
	- blockui 1.20.1-1.0.151-BETA
	- bookshelf 20.1.9
	- botania 1.20.1-443-FORGE
	- botany_pots_ore_planting 7.9.0
	- botanypots 13.0.25
	- botarium 2.3.3
	- caelus 3.1.0+1.20
	- chemlib 2.0.19
	- chunkpregen 1.20-4.4.2
	- chunksending 1.20.1-2.8
	- chunky 1.3.136
	- citadel 2.5.4
	- cloth_config 11.1.118
	- cofh_core 11.0.0
	- collective 7.40
	- commoncapabilities 2.9.1
	- computercraft 1.110.0
	- connectivity 1.20.1-5.5
	- cookingforblockheads 16.0.3
	- corgilib **.**.**.**
	- coroutil 1.20.1-1.3.7
	- crashutilities 8.1.4
	- create 0.5.1.f
	- create_astral_dimensions_compat 1.0.0
	- creativecore 2.11.25
	- cristellib 1.1.5
	- ctov 3.3.3
	- cucumber 7.0.7
	- cumulus_menus 1.20.1-1.0.0-neoforge
	- cupboard 1.20.1-2.6
	- curios 5.7.2+1.20.1
	- cyclopscore 1.19.0
	- davebuildingmod 5.0
	- decorative_blocks 4.1.0
	- domum_ornamentum 1.20.1-1.0.174-BETA
	- dsbg 1.0-1.20.1
	- dustrial_decor 1.3.2
	- edivadlib 2.0.1
	- electrodynamics 1.20.1-0.9.0-3
	- embeddium 0.3.9+mc1.20.1
	- epherolib 0.1.2
	- expandability 9.0.0
	- fastsuite 5.0.1
	- ferritecore 6.0.1
	- flib 0.0.11
	- flywheel 0.6.10-7
	- forge 47.2.20
	- forgeendertech **.**.**.**
	- formations 1.0.2
	- fps_optimizer 1.0.0
	- framework 0.6.27
	- ftblibrary 2001.1.5
	- fusion 1.1.1
	- geckolib 4.4.4
	- gunpowderlib 1.20.2-2.2.1
	- iceberg 1.1.21
	- irons_spellbooks 1.20.1-3.1.1
	- jei **.**.**.**
	- journeymap 5.9.18p1
	- kotlinforforge 4.10.0
	- kubejs 2001.6.4-build.138
	- libipn 4.0.1
	- libx 1.20.1-5.0.13
	- magistuarmory 9.2
	- many_more_music_discs 1.0
	- mcjtylib 1.20-8.0.3
	- mekanism 10.4.6
	- minecraft 1.20.1
	- mixinextras 0.2.0
	- mixinsquared 0.1.1
	- modonomicon 1.62.0
	- monolib 1.0.0
	- moonlight 1.20-2.11.9
	- mutil 6.1.1
	- mysterious_mountain_lib 1.4.4-1.20.1
	- nitrogen_internals 1.20.1-1.0.7-neoforge
	- nochatreports 1.20.1-v2.2.2
	- occultism 1.123.0
	- octolib 0.3
	- pamhc2foodcore 1.0.5
	- patchouli 1.20.1-84-FORGE
	- placebo 8.6.1
	- playeranimator 1.0.2-rc1+1.20
	- polylib 2000.0.3-build.132
	- prism 1.0.5
	- puzzlesaccessapi 8.0.7
	- puzzleslib 8.1.18
	- quarryplus 1201.0.51
	- reeses_sodium_options 1.0.4
	- resourcefulconfig 2.1.2
	- resourcefullib 2.1.23
	- rftoolsbase 1.20-5.0.2
	- rhino 2001.2.2-build.18
	- rubidium 0.7.1
	- searchables 1.0.2
	- sebastrnlib 4.0.0
	- securitycraft 1.9.9
	- servercore 1.3.9+1.20.1
	- shetiphiancore 1.20.1-1.2
	- silentlib 8.0.0
	- smartbrainlib 1.13
	- smoothboot 0.0.4
	- smoothchunk 1.20.1-3.6
	- sophisticatedcore **.**.**.**
	- structure_gel 2.16.1
	- supermartijn642configlib 1.1.8
	- supermartijn642corelib 1.1.17
	- terrablender **.**.**.**
	- textrues_embeddium_options 0.1.5+mc1.20.1
	- thermal 11.0.2
	- thermal_foundation 11.0.2
	- titanium 3.8.27
	- uteamcore **.**.**.**
	- valhelsia_core 1.1.2
	- watching 1.9.2
	- yungsapi 1.20-Forge-4.0.4
	- zerocore 1.20.1-2.1.41

### Logs

https://mclo.gs/ht3bKl9

### Minimal instance

- [ ] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by MehVahdJukaar** [comment]
could be an error from alex delight tho im not sure i see many many other errors there.

**Comment by Caldoric** [comment]
Ngl, this likely isn't going to be the last error like this that I stumble over... The final "modpack" I'll be running on this server is going to be upwards of 700 mods strong... Having to add them all in small chunks and check the run capability afterwards to see which mod(s) suddenly decide to throw a fit. Previously tried with all 700+ mods, and I got a lot of errors trying to start it up, so, doing chunk by chunk to fish out A) the mods that insist they're client-only, and B) the mods that run fine on client but somehow otherwise cause startup failure on the server.

The only reason the modlist is so large is because I combined the mod contents of two other modpacks from slightly older versions of Minecraft (one on 1.18, one on 1.19) and ended up adding a bunch of other fun and interesting mods I found along the way while trying to hunt down updated versions of everything from the other packs. Curseforge is both bane and boon in that regard...

Nonetheless, I doubt that there'll be any updates here on THAT topic. I'll have to check out the Alex's Delight possibility when I next get on my computer tho; I'll let y'all know about that part, at least.
